CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```

Comment:
Maybe we split in `### Feature`, `### Bug fixes`? Better than this crappy `### Changes`

```suggestion
## Unreleased

### Changes
```